test/e2e/config/development.json

```diff
@@ -1,4 +1,4 @@
 {
-	"calypsoBaseURL": "http://calypso.localhost:3000",
+	"calypsoBaseURL": "https://wordpress.com",
```

Comment:
Is this change necessary @sowmya2709 ?

Reply:
@kwight  -For some reason, I am not able to run calypso from my machine. So changed this, I will remove it in my final commit